pkg/chartverifier/checks/helm.go

```diff
@@ -229,4 +242,17 @@ func getImageReferences(chartUri string, vals map[string]interface{}) ([]string,
 	}
 
 	return images, err
+
+}
+
+func getNextLine(reader *bufio.Reader) (string, error) {
+	nextLine, isPrefix, err := reader.ReadLine()
+	if isPrefix && err == nil {
+		for isPrefix && err == nil {
+			var partLine []byte
+			partLine, isPrefix, err = reader.ReadLine()
+			nextLine = append(nextLine, partLine...)
+		}
+	}
+	return string(nextLine), err
```

Comment:
bufio.Reader already has a similar function:

```suggestion
	return reader.ReadString("\n")
```

Perhaps this function should be removed and bufio.Reader.ReadString() used in place?